packages/bot/src/stores/contract-card-store.js

```diff
@@ -0,0 +1,100 @@
+import {
+    observable,
+    action,
+    reaction }                from 'mobx';
+import { getIndicativePrice } from '../utils/contract';
+import { observer }           from '../utils/observer';
+
+export default class ContractCardStore {
+    @observable contract            = null;
+    @observable indicative_movement = '';
+    @observable profit_movement     = '';
+    @observable currency            = '';
```

Comment:
It seems like you didn't use this observable anywhere in `contract-card.jsx`. Can we remove this? If it's removable, we can remove `registerOnAccountSwitch` & `disposeOnAccountSwitch` as well